dli_app/templates/auth/resetpass.html

```diff
@@ -0,0 +1,24 @@
+{% extends 'layout.html' %}
+{% block body %}
+<div class="page-header">
+    <h1>Reset Password</h1>
+</div>
+
+<div class="col-md-6">
+    <form method="POST" action="{{ url_for('auth.resetpass', next='/') }}" class="form-horizontal">
```

Comment:
You don't need the next parameter. Your resetpass method redirects to home anyways.